airesources/CSharp/rungame.sh

```diff
@@ -5,16 +5,15 @@ rm -r *.hlt
 rm -r *.txt
 
 # start the dotnet build
-dotnet build
+dotnet build -c Release
 
 if [ $? -eq 0 ]
 then
 # run bots like it would do in our servers
-./halite -d "240 160" "dotnet ./bin/Debug/netcoreapp1.1/MyBot.dll" "python3 ../Python3/MyBot.py"
+./halite -d "240 160" "dotnet ./bin/Debug/netcoreapp2.0/Halite2.dll" "dotnet ./bin/Debug/netcoreapp2.0/Halite2.dll"
```

Comment:
Release in the build command but Debug in the ./halite command - looks a bit fishy?